lib/masamune/thor.rb

```diff
@@ -149,7 +149,7 @@ def initialize(_args = [], _options = {}, _config = {})
             end
           end
 
-          after_initialize_invoke(options.symbolize_keys)
+          after_initialize_invoke(options.to_hash.symbolize_keys)
```

Comment:
`Thor::HashWithIndifferentAccess` is behaving strangely after call to `symbolize_keys`- convert to plain old `Hash`